docs/cunumeric/source/user/usage.rst

```diff
@@ -0,0 +1,96 @@
+Usage
+=====
+
+Drop-in replacement
+-------------------
+
+Using cuNumeric as a replacement for NumPy is easy. Users only need
+to replace:
+
+.. code-block:: python
+
+  import numpy as np
+
+with
+
+.. code-block:: python
+
+  import cunumeric as np
+
+Running cuNumeric programs
+--------------------------
+
+Using ``legate``
+~~~~~~~~~~~~~~~~
+
+The preferred way to run cuNumeric programs is with the ``legate`` driver:
+
+.. code-block:: sh
+
+  legate --cpus 2 --gpus 2 script.py <script options>
+
+All ``legate`` command line configuration options must come *before* the script
+that is be executed. Any options after the script are passed on to the script.
+
+It is also possible to pass the ``legate`` command line configuration options
+using the ``LEGATE_CONFIG`` environment variable:
+
+.. code-block:: sh
+
+  LEGATE_CONFIG="--cpus 2 --gpus 2" legate script.py <script options>
+
+Using standard Python
+~~~~~~~~~~~~~~~~~~~~~
+
+It is also possible to run cuNumeric programs using the standard Python
+interpreter, with some limitations:
+
+.. code-block:: sh
+
+  LEGATE_CONFIG="--cpus 2 --gpus 2" python script.py <script options>
+
+When running programs with this method, configuration options may only be
+passed via the ``LEGATE_CONFIG`` environment variable as shown above.
+
+Additionally, several ``legate`` command line configuration options are not
+available when running programs this way. See the output of ``legate --help``
+for more details.
+
+Configuration options
```

Comment:
Is this section empty by design?

Reply:
I moved config to an entire separate chapter, this heading was removed in https://github.com/nv-legate/cunumeric/pull/805/commits/4f1b1f971576178da50a99ef5c28f2711d64a6b2